backend/btrixcloud/models.py

```diff
@@ -977,16 +988,90 @@ class OrgQuotas(BaseModel):
 
 
 # ============================================================================
-class OrgCreate(BaseModel):
-    """Create a new org"""
+class SubscriptionCreate(BaseModel):
+    """create new subscription"""
 
-    name: Optional[str] = None
-    slug: Optional[str] = None
+    type: Literal["create"] = "create"
 
-    firstAdminInviteEmail: Optional[str] = None
+    subId: str
+    status: str
+    planId: str
+
+    firstAdminInviteEmail: str
     quotas: Optional[OrgQuotas] = None
 
-    subData: Optional[Dict[str, Any]] = None
+
+# ============================================================================
+class SubscriptionUpdate(BaseModel):
+    """update subscription data"""
+
+    type: Literal["update"] = "update"
+
+    subId: str
+    status: str
+    planId: str
```

Comment:
Is the value of this going to be the name of the plan, or are we keeping a mapping somewhere else of plan ids -> names? If the former, maybe just name it `plan` or `planName`?

Reply:
Yes, the idea is that it would be an id that could be mapped to a name on the frontend - if it is a name, that makes localization trickier.